tests/test_lab_download.py

```diff
@@ -16,15 +16,15 @@ class TestLabDownload:
     # of your module, so you should use `my_module.Y`` to patch.
     # When using `import X`, you should use `X.Y` to patch.
     # https://docs.python.org/3/library/unittest.mock.html#where-to-patch?
-    @patch("instructlab.lab.hf_hub_download")
+    @patch("instructlab.model.download.hf_hub_download")
     def test_download(self, mock_hf_hub_download):
         runner = CliRunner()
         with runner.isolated_filesystem():
             result = runner.invoke(
                 lab.ilab,
                 [
                     "--config=DEFAULT",
-                    "download",
+                    "model" "download",
```

Comment:
This is missing a command between "model" and "download".

I'm assuming the test is failing. I did see the test CI job is failing